### PR TITLE
[fix] fixed indentation of default-cr.yml

### PR DESCRIPTION
**Pull request by mmuzikar**
publicApi key had wrong indentation which failed the whole installation

### CODE REVIEW

**Comment by mmuzikar** [comment]
@heiko-braun can you get someone to review this?

File: default-cr.yml [hunk 1]
```diff
@@ -87,7 +87,7 @@ spec:
     ops:
       enabled: false
     # Exposes public REST API endpoints for CI/CD pipelines.
-   publicApi:
+    publicApi:
        enabled: false
        # Uncomment & modify to specify the public address for invoking REST API endpoints.
        routeHostname: public-syndesis.a.b.c.d.e
```
Comment (zregvart) [inline default-cr.yml]: Should this also be aligned?

```suggestion
      enabled: false
      # Uncomment & modify to specify the public address for invoking REST API endpoints.
      routeHostname: public-syndesis.a.b.c.d.e
```

Comment (heiko-braun) [reply default-cr.yml]: I guess you are right. my bad, I did merge it eagerly

Comment (heiko-braun) [reply default-cr.yml]: Here's a followup: https://github.com/syndesisio/fuse-online-install/pull/226

**Pull request merged**